src/components/registration/RegistrationTable.jsx

```diff
@@ -42,7 +41,14 @@ function getTableRows(families, extraFields) {
 
 function getTableData(families, extraColumns) {
   const rows = getTableRows(families, extraColumns);
-  const columns = [];
+  const columns = [
+    {
+      name: "id",
+      options: {
+        display: "excluded",
+      },
+    },
+  ];
```

Comment:
Not sure if this is the best way to add the `id` column 😅

Reply:
the name & options look good! can we add this to `src/constants/DefaultFields.js`?